charts/agent-control-deployment/templates/deployment-agentcontrol.yaml

```diff
@@ -116,6 +116,20 @@ spec:
 
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
```

Comment:
what's about readiness probe? We do not receive traffic, but could it be useful anyway?

Reply:
I started with livenessProbe + readinessProbe but I switched to startupProbe + livenessProbe. I'll double check and share the explanation